Documentation/Introduction/Index.rst

```diff
@@ -13,84 +13,80 @@ Introduction
+and fail-safe. Most TYPO3 sites are dependent on a number of extensions. Some
+examples are "News" or "Powermail". A sitepackage extension which contains
+global configuration variables for these extensions will define the dependencies
+for you. When the sitepackage extension is installed in an
+empty TYPO3 instance, all dependent extensions are automatically downloaded from
+the `TYPO3 Extension Repository <https://extensions.typo3.org>`__ and installed.
 
 Clean Separation from the Userspace
 -----------------------------------
-Without using an extension, template files are often stored in the
-:file:`fileadmin/` directory of a TYPO3 instance. As a matter of fact, files in
-this directory are indexed by TYPO3's File Abstraction Layer (FAL). This
-results in irrelevant and avoidable records in the database. From a logical
-perspective, the :file:`fileadmin/` area is the "userspace". This area is meant to
-belong to editors. Even if access permissions restrict editors from accessing
-or manipulating files in :file:`fileadmin/`, site configuration components should
-not be stored in the userspace by all means.
+In a TYPO3 installation that doesn't use extensions, template files are often
+stored in the :file:`fileadmin/` directory. Files in
+this directory are indexed by TYPO3's File Abstraction Layer (FAL) resulting in
+possibly irrelevant records in the database. To avoid this the :file:`fileadmin/`
+area should be seen as a "userspace" which is only available for editors to
+use. Even if access permissions restrict editors from accessing or manipulating
+files in :file:`fileadmin/`, site configuration components should
+still not be stored in the userspace.
 
 Security
 --------
-Files in :file:`fileadmin/` are typically meant to be publicly accessible per
+Files in :file:`fileadmin/` are typically meant to be publicly accessible by
 convention. To avoid disclosing sensitive system information (see the
 :ref:`TYPO3 Security Guide <t3coreapi:security>` for further details),
 configuration files should not be stored in :file:`fileadmin/`.
 
 Deployment
 ----------
-Modern versions of TYPO3 CMS follow the *convention over configuration*
-paradigm. As a consequence, if files and directories of the sitepackage
-extension use a specific naming convention, they are loaded automatically as
-soon as the extension is installed/activated in the system. This leads to
-another important advantage of the sitepackage extension. The extension can be
-deployed easily by using the Extension Manager and/or the PHP composer method.
-This also enables system administrators to automate the deployment for example.
+The TYPO3 CMS follows the *convention over configuration*
+paradigm. If files and directories in the sitepackage
+extension use the naming convention, they are loaded automatically as
+soon as the extension is installed/activated. This means the
+extension can be easily deployed using the Extension Manager and/or PHP composer.
```

Comment:
Don't understand the sentence. "deployed using the Extension Manager". What is "PHP Composer"?

I know, this was already so, maybe leave it for now and tackle this in another patch.